go/chat/identify_ui.go

```diff
@@ -19,17 +21,27 @@ func (c chatNullIdentifyUI) FinishSocialProofCheck(libkb.MetaContext, keybase1.R
 func (c chatNullIdentifyUI) Confirm(libkb.MetaContext, *keybase1.IdentifyOutcome) (keybase1.ConfirmResult, error) {
 	return keybase1.ConfirmResult{}, nil
 }
-func (c chatNullIdentifyUI) DisplayCryptocurrency(libkb.MetaContext, keybase1.Cryptocurrency) error          { return nil }
-func (c chatNullIdentifyUI) DisplayStellarAccount(libkb.MetaContext, keybase1.StellarAccount) error          { return nil }
-func (c chatNullIdentifyUI) DisplayKey(libkb.MetaContext, keybase1.IdentifyKey) error                        { return nil }
-func (c chatNullIdentifyUI) ReportLastTrack(libkb.MetaContext, *keybase1.TrackSummary) error                 { return nil }
-func (c chatNullIdentifyUI) LaunchNetworkChecks(libkb.MetaContext, *keybase1.Identity, *keybase1.User) error { return nil }
-func (c chatNullIdentifyUI) DisplayTrackStatement(libkb.MetaContext, string) error                           { return nil }
-func (c chatNullIdentifyUI) DisplayUserCard(libkb.MetaContext, keybase1.UserCard) error                      { return nil }
-func (c chatNullIdentifyUI) ReportTrackToken(libkb.MetaContext, keybase1.TrackToken) error                   { return nil }
-func (c chatNullIdentifyUI) Cancel(libkb.MetaContext) error                                                { return nil }
-func (c chatNullIdentifyUI) Finish(libkb.MetaContext) error                                                { return nil }
+func (c chatNullIdentifyUI) DisplayCryptocurrency(libkb.MetaContext, keybase1.Cryptocurrency) error {
+	return nil
```

Comment:
is there some linter fight going on here? i can recall some other pr changed it the other way a while ago